docs/Agent/user-guide/guides/end-to-end-workflow.md

```diff
@@ -0,0 +1,21 @@
+### End-to-End Workflow
+
+Each command in pbench-agent accepts the `--help` option and outputs a brief usage message
+
+The default set of tools for data collection can be enabled with
+
+	$pbench-register-tool-set
```

Comment:
While the very simple default is great for a first-level overview, using the MarkDown link syntax to provide a link to full details on using `pbench-register-tool-set` (and the other commands) would be a great way to dig into the documentation tree. (The targets don't have to exist yet, though having at least placeholders would be nice.)